test/TestAsset/Check/SecurityAdvisory.php

```diff
@@ -9,15 +9,15 @@
 namespace LaminasTest\Diagnostics\TestAsset\Check;
 
 use Laminas\Diagnostics\Check\SecurityAdvisory as BaseCheck;
-use SensioLabs\Security\SecurityChecker;
+use Enlightn\SecurityChecker\AdvisoryAnalyzer;
 
 class SecurityAdvisory extends BaseCheck
 {
     /**
-     * @param SecurityChecker $securityChecker
+     * @param \Enlightn\SecurityChecker\AdvisoryAnalyzer $advisoryAnalyzer
      */
-    public function setSecurityChecker(SecurityChecker $securityChecker)
+    public function setAdvisoryAnalyzer(AdvisoryAnalyzer $advisoryAnalyzer)
```

Comment:
Overall LGTM, but signature changes require a major version bump :-\

Reply:
Hmm, I see. I guess there's no way around it given the Sensiolabs SecurityChecker type hinted class. Does that mean this PR needs to wait?

Reply:
@weierophinney probably need your input here

Reply:
Could an exception be made seeing as the SecurityChecker no longer works?

Reply:
The service itself issues a warning when invoked, don't see a need for throwing an exception.
> WARNING Security Advisory: The web service failed for an unknown reason (HTTP 403).

Reply:
I misread this: this is a test asset, so no major release needed.